core/qt-ble.cpp

```diff
@@ -65,21 +68,24 @@ void BLEObject::characteristcStateChanged(const QLowEnergyCharacteristic &c, con
 {
 	if (IS_HW(device)) {
 		if (c.uuid() == hwAllCharacteristics[HW_OSTC_BLE_DATA_TX]) {
+			hw_credit--;
 			receivedPackets.append(value);
+			if (hw_credit == MINIMAL_HW_CREDIT)
+				setHwCredit(MAXIMAL_HW_CREDIT - hw_credit);
```

Comment:
since you test for '==' in the line above, this is actually the same as MAX-MIN (or a fixed 223)